Codecademy/gamut
# feat(EditableIcon): Add YoutubePlayIcon

### Overview



Add YoutubePlayIcon



<img width="239" alt="Screenshot 2024-04-23 at 5 49 50 PM" src="https://github.com/Codecademy/gamut/assets/14988121/58f52786-8fe9-4f3e-a189-d5a763ea642b">

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/7Oe7eNRzW9J0KbShrfbiPy/Events?type=design&node-id=2876-30977&mode=design&t=YG6b1howjjfcM7hh-4
- [x] Related to JIRA ticket: [LEGO-3809](https://codecademy.atlassian.net/browse/LEGO-3809)

#### Testing Instructions



Follow the testing instructions in the monorepo PR to see the icon being used on event cards.

Check that the icon looks good in Storybook: https://66282b85db395b1e4f1aa7a0--gamut-preview.netlify.app/?path=/docs/atoms-icons-regular--editable-icon

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/5964)  | [Portal Env](https://tengu.codecademy.com/events?PR_ENV=portal-app-pr-5964)   |




[LEGO-3809]: https://codecademy.atlassian.net/browse/LEGO-3809?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
